src/credentials/BUILD.gn

```diff
@@ -126,6 +128,23 @@ static_library("credentials") {
   ]
 }
 
+source_set("test_paa_store") {
+  sources = [
+    "attestation_verifier/CHIPAttCert_test_paa_store.cpp",
+    "attestation_verifier/CHIPAttCert_test_paa_store.h",
+  ]
+
+  public_deps = [
+    "${chip_root}/src/lib/core:chip_config_header",  # for lib/support/Span.h
+    "${nlassert_root}:nlassert",  # for lib/support/Span.h
```

Comment:
Followup: Span.h should actually be in a source_set that has the right deps, so consumers don't have to do this.

Reply:
darwin is the problem here - I think I just force updated, but I did have it as its own source set, and there's a circular dependency between span and the darwin logging that I cannot figure out how to untangle.

Reply:
Especially since I have to push it through the CI to compile darwin.

Reply:
Right, that's why I said "Followup"!  Please just file the issue.  If you have a diff that works except for Darwin, please point me to it and I will sort out the Darwin bits.

Reply:
@cecille PR up at https://github.com/project-chip/connectedhomeip/pull/29491